### PR TITLE
Fix selected python rosdep keys for Xenial.

### PR DESCRIPTION
**Pull request by mikepurvis**
Affected: python-termcolor, python-enum, python-pandas, python-psycopg2, python-avahi, python-sqlalchemy

### CODE REVIEW

File: rosdep/python.yaml [hunk 1]
```diff
@@ -582,12 +571,7 @@ python-empy:
     xenial: [python-empy]
 python-enum:
   fedora: [python-enum]
-  ubuntu:
-    precise: [python-enum]
-    quantal: [python-enum]
-    raring: [python-enum]
-    saucy: [python-enum]
-    trusty: [python-enum]
+  ubuntu: [python-enum]
```
Comment (mikepurvis) [inline rosdep/python.yaml]: Possible that on Xenial, this should resolve to python-enum34, given that the two packages are now in conflict.

File: rosdep/python.yaml [hunk 2]
```diff
@@ -2441,7 +2398,12 @@ python-vtk:
   arch: [vtk]
   debian: [python-vtk]
   fedora: [vtk-python]
-  ubuntu: [python-vtk]
+  ubuntu:
+    trusty: [python-vtk]
+    utopic: [python-vtk]
+    vivid: [python-vtk]
+    wily: [python-vtk]
+    xenial: [python-vtk6]
```
Comment (mikepurvis) [inline rosdep/python.yaml]: I believe this is what should happen, given that `python-vtk` and `python-vtk6` are in conflict, and the base `libvtk` rosdep in Xenial points to `libvtk6-dev`, which pulls in `python-vtk6`:

https://github.com/ros/rosdistro/blob/ec21789bdf300a7ec4e21a1acbd391caa9b97447/rosdep/base.yaml#L2749
http://packages.ubuntu.com/xenial/libvtk6-dev

**Comment by mikepurvis** [comment]
Would appreciate getting this merged before there end up being conflicts with it.

@dirk-thomas @wjwwood @tfoote

**Comment by dirk-thomas** [comment]
For the keys where you added a `xenial` entry please also add all in-between releases, e.g. `wily`.

Beside this comment +1

**Comment by tfoote** [comment]
@mikepurvis can you fill in the wily entry for python-termcolor?

**Comment by mikepurvis** [comment]
Done, sorry about that.

**Pull request merged**